# Super simple estimation of available solar energy
> How much energy is actually renewable and humanity's cap in consumption

- toc: true 
- badges: true
- comments: true
- categories: [energy]

# Solar energy

## Stefan boltzmann's law

$ \text{Surface energy} = \sigma T^4$

For the sun, $T = \text{5,778 }K$

$\sigma = 5.67 \times 10 ^{-8} W.m^{-2}.K^{-4}$


In [6]:
from sympy.physics.units import K, W, m

sigma = 5.67 * 10**(-8) * W *m**(-2) * K**(-4)
T = 5778 * K
surface_energy = sigma * T**4
print(surface_energy)

63196526.5460292*watt/meter**2


##  Total emitted solar energy

$ Radiation = \text{Surface of the sun} \times \text{Surface energy} $


$ Radiation = 4 \pi r^2 \times \text{Surface energy} $


In [7]:
from sympy import *

r_sun = 696_340 * 1000 *m
surface_of_sun = 4 * pi * r_sun ** 2 
radiation = surface_of_sun * surface_energy

print(radiation)

1.22573302243694e+26*pi*watt


## Energy received at earth average distance

$ \text{Radiation received} = \frac{\text{Total sun radiation}}{ \text{sphere at earth's distance}}$

$ \text{Radiation received} = \frac{Radiation}{ 4 \pi D_{earth-sun}^2} $

In [8]:
R_earth = 6_371 * 1000 * m
D_earth_sun = 148.88 * 10**6 * 1000 * m
earth_perp_surface = pi * R_earth **2
sphere = 4 * pi * D_earth_sun **2

radiation_received = radiation / sphere
print(radiation_received)

1382.49374484614*watt/meter**2


## Energy received by the earth surface (before atmosphere)

$ \text{Energy received} = \text{radiation received} \times \frac{ \text{visible surface}}{ \text{earth's surface}} $

In [9]:
power_received = radiation_received * pi * R_earth **2
surface_power_received = power_received / (4 * pi * R_earth **2)

print(surface_power_received)
print(power_received.n())

345.623436211536*watt/meter**2
1.76290235470883e+17*watt


> RADIATION RECEIVED BY SYSTEM EARTH  = $345 W.m^{-2}$
>
> MAXIMUM POWER WITH EARTH "DYSON SPHERE": $176 PW$

# Human consumption

13 511 MTep [Source International Energy agency](https://www.iea.org/data-and-statistics?country=WORLD&fuel=Energy%20supply&indicator=Total%20primary%20energy%20supply%20%28TPES%29%20by%20source)





In [10]:
from sympy.physics.units import J
tep = 41_868 *10**9 * J
Mtep = 10**6 * tep
Humanity_total_annual_consumption = 13_511 * Mtep

print(Humanity_total_annual_consumption)

565678548000000000000000*joule


In [11]:
from sympy.physics.units import s
from sympy.physics.units.util import convert_to
humanity_power_consumption = Humanity_total_annual_consumption / (365.25 * 24 * 60 * 60 * s)
print(convert_to(humanity_power_consumption, [W]))

1.79252715035364e+16*watt


In [12]:
from sympy.physics.units.util import convert_to
print(convert_to(humanity_power_consumption / power_received, [J, s]).n())

0.101680455844062


We are currently consuming **10% of the maximum capacity of the earth covered by a Dyson sphere of solar panels**.

### A bit more realistic approach

After the atmosphere only $168 W.m^{-2}$ hit the surface. It's quite complicated to infer it depends on the wavelength of the incoming light, clouds, composition of the atmosphere and so on, so we just take the value from [here](https://fr.wikipedia.org/wiki/Bilan_radiatif_de_la_Terre).

Then we only have 29% of the earth surface that is landmass (where we can reasonably put solar panels in large quantity)

Of that 31% is covered in forest which are already some natural solar panels we don't  want to remove (for other obvious reasons) [source](http://www.earth-policy.org/indicators/C56/forests_2012)
And 38.4% is covered of agricultural land [source](https://en.wikipedia.org/wiki/Agricultural_land).

Then solar panels are not 100% efficient. They are roughly only 20% efficient with current technology at a reasonable cost.

In [13]:
earth_power_received = 168 * W * m **(-2)

available_surface = 4 * pi * R_earth **2 * 0.29 * (1 -.31 - .384)

max_power = earth_power_received * available_surface * 0.2

print(max_power.n())
print(convert_to(humanity_power_consumption / max_power *100, [J, s]).n())

1.52084087357243e+15*watt
1178.64214560661


# Conclusion

In the end we are currently consuming **1179% of the realistic available solar power energy**. That's 10 times more than we should. And 100% would require posing solar panels everywhere on the planet that is not a forest or agricultural land.

NB: This is a very superficial attempt to evaluate these numbers, however the result should be correct within an order of magnitude.